erdpy/testnet/setup.py

```diff
@@ -268,6 +268,6 @@ def build_binaries(testnet_config: TestnetConfiguration):
 def _get_wasm_vm_version(testnet_config: TestnetConfiguration):
     go_mod = testnet_config.node_source() / "go.mod"
     lines = utils.read_lines(go_mod)
-    line = next(line for line in lines if "github.com/ElrondNetwork/arwen-wasm-vm" in line)
+    line = next(line for line in lines if "github.com/ElrondNetwork/wasm-vm" in line)
```

Comment:
[optional] should maybe also switch to the new repo?

Reply:
Unfortunately, this depends on go.mod:

https://github.com/multiversx/mx-chain-go/blob/master/go.mod

Thus, we have to actually use the oldest name (please revert this change).

Reply:
Reverted to old name